tests/testmodels.py

```diff
@@ -240,36 +261,50 @@ class ImplicitPkModel(Model):
 class UUIDPkModel(Model):
     id = fields.UUIDField(pk=True)
 
+    children: fields.RelationQueryContainer["UUIDFkRelatedModel"]
+    children_null: fields.RelationQueryContainer["UUIDFkRelatedNullModel"]
+    peers: fields.ManyToManyRelationManager["UUIDM2MRelatedModel"]
```

Comment:
You definitely needn't be so thorough here in the test models. We probably will have to deal with a crazy mix of complete and incomplete annotations.

I would take it out of the UUIDPkModel and its related, but leave it in the UUIDPkSourceModel and related. (or vice versa)
Reason for this that these two sets of models are run by the exact same test cases each, and we want to ensure that the differences in specifying models don't affect actual API usage at all.

Reply:
Got it. I left only `UUIDPkModel`.